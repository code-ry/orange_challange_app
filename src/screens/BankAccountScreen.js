import React from 'react'
import { StyleSheet, Text, View } from 'react-native'

const BankAccountScreen = () => {
    return (
        <View>
            <Text>Bank Accounts</Text>
        </View>
    )
}

export default BankAccountScreen

const styles = StyleSheet.create({})
