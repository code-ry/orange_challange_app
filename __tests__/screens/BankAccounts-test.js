import React from 'react';
import renderer from 'react-test-renderer';
import BankAccountScreen from '../../src/screens/BankAccountScreen';

test('renders correctly', () => {
  const tree = renderer.create(<BankAccountScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});