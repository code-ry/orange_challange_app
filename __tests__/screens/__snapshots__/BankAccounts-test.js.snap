// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text>
    Bank Accounts
  </Text>
</View>
`;
